client/state/data-layer/wpcom/activity-log/rewind/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchRewindStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
+	}, action ) );
+};
+
+const fromApi = response => ( {
+	active: response.use_rewind,
+	firstBackup: response.first_backup_when,
+	error: { message: response.error },
+} );
+
+const updateRewindStatus = ( { dispatch }, { siteId }, next, data ) => {
+	dispatch( {
+		type: REWIND_STATUS_REQUEST,
```

Comment:
Because as-is we are telling Calypso "the way to respond to a request to get data X is that after we get data X we should request to get data X" it is an infinite loop because we have told it to be so